plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -433,11 +435,9 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
       void switch_to_read_window();
```

Comment:
Should `_ro_trx_queue` be renamed to something like `_ro_exhausted_read_only_trx_queue` as now it does not store regular read-only transactions?

Reply:
Renamed